test/src/runner/testRunWithVerify.js

```diff
@@ -27,9 +27,9 @@ describe('testRunWithVerify', function() {
     CommandGlobals.afterEach.call(this, done);
   });
 
-  it('using verify in tests', function() {
+  it('using verify in tests with simple failures', function() {
     return runTests({
-      _source: ['./withverify']
+      _source: ['./withverify/verifySampleFailures.js']
```

Comment:
We need absolute paths here, as it is the case with all other runner tests. Please have a look at the other tests and use the same pattern.